src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj

```diff
@@ -6,6 +6,14 @@
     <WasmMainJSPath>main.js</WasmMainJSPath>
     <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <EnableAOTAndTrimming>true</EnableAOTAndTrimming>
```

Comment:
@radekdoulik would make your perf testing all AOT ?

Reply:
ah, good point, the other projects I did this to had AOT on originally but this didn't. I'll change it back.